demosplan/DemosPlanCoreBundle/ResourceTypes/StatementResourceType.php

```diff
@@ -276,6 +277,20 @@ protected function getProperties(): array|ResourceConfigBuilderInterface
 
                     return '' === $draftsListJson ? null : Json::decodeToArray($draftsListJson);
                 });
+            $configBuilder->status->readable(true, function (Statement $statement) {
+                if ($statement->isAlreadySegmented()) {
```

Comment:
I thought we wanted to move that logic to a centralized place, that it can be used by the RPC-request too?!?